classes/01class/exercises/c01-e01/andreazzabr/c01-e01.txt

```diff
@@ -0,0 +1,20 @@
+anderson@andreazza:~$ aws --version
+aws-cli/1.16.266 Python/3.5.2 Linux/5.3.0-28-generic botocore/1.13.2
+
+
+anderson@andreazza:~$ docker version 
+Client:
+ Version:           18.09.9
+ API version:       1.39
+ Go version:        go1.13.4
+ Git commit:        1752eb3
+ Built:             Sat Nov 16 01:05:26 2019
+ OS/Arch:           linux/amd64
+ Experimental:      false
+Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.39/version: dial unix /var/run/docker.sock: connect: permission denied
```

Comment:
did you install docker as root on linux? what tutorial did you follow? Don't use the one from package managers.

Follow this tutorial:

https://docs.docker.com/install/linux/docker-ce/centos/

Then update your commit with no showing error. That error won't allow you to run containers.

Reply:
Or the last step of the link provided by @caiocezart was not completed. You need to include your user to the docker group:
` sudo usermod -aG docker your-user`

Reply:
@caiocezart  I've been trying to reinstall docker following the tutorial you mentioned, but seems like australian archive is offline the whole day. I'll try again tomorrow. (I installed it using Ubuntu Software)
PS1: if i do "sudo docker version", i don't get the permission error
PS2: @kikobr82  i couldn't find the group "docker" to add my user, I'll try again tomorrow after installing with apt-get.

Err:4 http://au.archive.ubuntu.com/ubuntu bionic InRelease                                                                                                             
  Cannot initiate the connection to au.archive.ubuntu.com:80 (2001:388:30bc:cafe::beef). - connect (101: Network is unreachable) Could not connect to au.archive.ubuntu.com:80 (202.158.214.106), connection timed out